doc/whats_new/v0.20.rst

```diff
@@ -851,6 +851,13 @@ Preprocessing
   will be from ``True`` to ``False`` in 0.22.
   :issue:`10655` by :user:`Guillaume Lemaitre <glemaitre>`.
 
+Model selection
+
+- The default number of cross-validation folds ``cv`` and the default number of
+  splits ``n_splits`` in the :class:`model_selection.KFold`-like splitters will change
+  from 3 to 5 in 0.22 as 3-fold has a lot of variance.
+  :issue:`11129` by :user:`Alexandre Boucaud <aboucaud>`.
+
```

Comment:
should be the number of the PR not the issue, right ?

Reply:
dunno, you tell me sprint master.

Reply:
confirmed